### ISSUE TITLE
Update build.md with working instructions

### ISSUE THREAD

**Issue by ByteTemplar** [Androxyde/Flashtool#73]
At the moment, following building instructions, the resulted .JAR file only has manifest. Its obvious something is missing. Documentation is very important to keep up to date! 

Im trying to build it myself, cause the download webpage has mismatching MD5 value and this raises questions!

**Comment by derMart** [comment]
Yep, currently there is no build process to compule x10flasher.jar
Also mentioned here https://github.com/Androxyde/Flashtool/issues/54
@Androxyde could you have a look at the build scripts?
